strapi/strapi
# Fix transfer DZ and repeatable components

### What does it do?

filters out links related to components so we don't export them

### Why is it needed?

- components and dz are already populated at the entity stage 
- we face a unique constraint error otherwise by attempting to insert duplicate ids

### How to test it?

- export a strapi app with repeatable components and dz data it should work if you try to import it back

fix #20394
